.github/workflows/cmake_tests.yml

```
@@ -0,0 +1,39 @@
+name: CMake build
+
+on:
+  workflow_call:
+    inputs:
+      update_cmake_lists_config:
+        type: string
+        description: "The configuration used when updating the CMake lists."
+        required: true
+      cmake_build_target_directory:
+        type: string
+        description: "The directory to pass to `cmake build`."
+        default: "."
+      cmake_version:
+        type: string
+        description: "The version of CMake to install."
+        default: ""
+
+jobs:
+  cmake-checks:
+    name: CMake checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          submodules: true
+      - name: Mark the workspace as safe
+        # https://github.com/actions/checkout/issues/766
+        run: git config --global --add safe.directory ${GITHUB_WORKSPACE}
+      - name: Check CMake lists
```

This irks me. These are referred to as "CMakeLists files" in the docs: https://cmake.org/cmake/help/book/mastering-cmake/chapter/Writing%20CMakeLists%20Files.html

---

So you'd rather this said the following?
```suggestion
      - name: Check CMakeLists files
```